interp/modintern.ml

```diff
@@ -59,33 +59,42 @@ let lookup_module lqid = fst (lookup_module_or_modtype Module lqid)
 
 let transl_with_decl env = function
   | CWith_Module ((_,fqid),qid) ->
-      WithMod (fqid,lookup_module qid)
+      WithMod (fqid,lookup_module qid), Univ.ContextSet.empty
   | CWith_Definition ((_,fqid),c) ->
     let c, ectx = interp_constr env (Evd.from_env env) c in
     let ctx = UState.context ectx in
```

Comment:
This should be in the polymorphic branch, so as to not linearize for nothing when monomorphic.

Reply:
WDYM?